sbt

```diff
@@ -0,0 +1,568 @@
+#!/usr/bin/env bash
```

Comment:
Why this file is related with this PR?

Reply:
this is a convince helper, to avoid installing another random version of sbt

Reply:
so, is not related, right? could you do another PR to add it please? We should separate concepts as much as possible to made reviews easy.